app/views/layouts/madeline/template.html.erb

```diff
@@ -0,0 +1,18 @@
+<%# Defines content template variables based on page displayed %>
+<%= render "layouts/madeline/page_variables" %>
+
+<!doctype html>
+<html lang="en-US">
+
+  <head>
+    <meta charset="UTF-8">
+    <title><%= yield :title %> | Madeline System</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <%= yield :head %>
+  </head>
+
+  <body>
+    <%= render @content_template %>
+  </body>
+
+</html>
```

Comment:
I'm still not sure why all this isn't just in application.html.slim. Want to talk about it tomorrow?